components/form-file-upload/editor.scss

```diff
@@ -0,0 +1,11 @@
+.components-form-file-upload {
```

Comment:
The file should be `style.scss` not `editor`; `editor.scss` is used to differentiate editor styles for _blocks_.

Reply:
I see. It makes sense, but it's a bit confusing. I assumed that it needs to be editor view as this is where it only applies :)

Reply:
The editor / style split is only for blocks, as we need to load those on the front end, and we don't want to pull block styles that are meant for the editor.

Reply:
Thanks for making it clear :) It's now updated to match the convention.